torch_geometric/loader/utils.py

```diff
@@ -24,6 +25,22 @@ def index_select(value: Tensor, index: Tensor, dim: int = 0) -> Tensor:
     return torch.index_select(value, 0, index, out=out)
 
 
+def unbatcher(src: torch.tensor, index: torch.tensor, dim: int = -1) -> list:
+    scatter = pandas.DataFrame(
+        index, columns=['scatter']).reset_index().groupby('scatter').tail(1)
+    indices = []
+    index_i = 0  # index starts for a graph
+    index_f = 0  # index ends for a graph
+    for i in range(0, len(scatter)):
```

Comment:
Any chance we can remove the (expensive) for-loop via by making use of vectorized operations?

Reply:
At this moment I don't have any more efficient way, this loop find all node_attrs or edge_attrs for all graphs in the batch. We then use ```Data``` and ```to_networkx``` to reconstruct all graphs.